.github/workflows/check-extension-case.yml

```diff
@@ -26,7 +26,7 @@ jobs:
               do
                   fileExtension=$(echo $filename | sed 's/"//g' | sed 's/]//g' | grep -o '...$')
                   lowercaseFileExtension=$(echo "$fileExtension" | tr '[:upper:]' '[:lower:]')
-                  if [ $lowercaseFileExtension == 'gif' ] || [ $lowercaseFileExtension == 'svg' ]
+                  if [ $lowercaseFileExtension == 'gif' ] || [ $lowercaseFileExtension == 'svg' ] [ $lowercaseFileExtension == 'png' ] || [ $lowercaseFileExtension == 'jpg' ]
```

Comment:
Do we support any other image files where we could also have this issue?

Reply:
Yes, SSW.Rules.Content repo has `.webp` and `.jpeg` files, so I've added check for these two file types ✅